src/routes/PersonDetails.tsx

```diff
@@ -0,0 +1,190 @@
+                        />
+                    )}
+                    <div className="pt-2 text-center">
+                        <p className="font-bold">{contactName}</p>
+                        <p className="text-xs text-gray-200">
+                            {contactPhoneNumber}
+                        </p>
+                    </div>
+                </div>
+
+                <div className="mt-6 flex w-full justify-between">
+                    <div className="flex flex-col">
+                        <div className="rounded-full bg-white p-3 text-purple-800">
+                            <Envelope width={28} height={28} className="" />
+                        </div>
+                        <p className="pt-2 text-center text-xs">Message</p>
+                    </div>
+                    <div>
+                        <div className="rounded-full bg-white p-3 text-purple-800">
+                            <Call width={28} height={28} className="" />
+                        </div>
+                        <p className="pt-2 text-center text-xs">Call</p>
+                    </div>
+                    <div>
+                        <div className="rounded-full bg-white p-3 text-purple-800">
+                            <Webcam width={28} height={28} className="" />
+                        </div>
+                        <p className="pt-2 text-center text-xs">Video</p>
+                    </div>
+                    <div>
+                        <div className="rounded-full bg-white p-3 text-purple-800">
+                            <Share width={28} height={28} className="" />
+                        </div>
+                        <p className="pt-2 text-center text-xs">Share</p>
+                    </div>
+                </div>
+            </div>
+
+            <div className="flex flex-grow flex-col rounded-3xl bg-white px-6 py-4">
+                <div className="border-b border-slate-200 py-2">
+                    <p className="text-sm font-black">Mobile</p>
+                    <p className="text-xs text-slate-400">
+                        {contactPhoneNumber}
+                    </p>
+                </div>
+                <div className="border-b border-slate-200 py-2">
+                    <p className="text-sm font-black">Group</p>
+                    <p className="text-xs text-slate-400">{contactGroup}</p>
+                </div>
+                <div className="border-b border-slate-200 py-2">
+                    <p className="text-sm font-black">Link numbers</p>
+                    <p className="text-xs text-slate-400">{contactGroup}</p>
+                </div>
+                <div className="border-b border-slate-200 py-2">
+                    <p className="text-sm font-black">Add to favorites</p>
+                </div>
+                <div className="border-b border-slate-200 py-2">
+                    <p className="text-sm font-black">Share my location</p>
+                </div>
```

Comment:
if these have all the same styles you could make a static array and map through it